src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -5697,7 +5698,7 @@ public void IOFailure_MovePdbTemp()
             MockCSharpCompiler csc = MakeTrackingCsc(tempFilePaths, "/nologo", "/debug", "/preferreduilang:en", sourcePath);
             csc.FileMove = (source, dest) =>
             {
-                if (dest.EndsWith(".pdb"))
+                if (dest.EndsWith(".pdb", StringComparison.OrdinalIgnoreCase))
```

Comment:
Should we define a `StringComparer` / `StringComparison` which is ordinal for *nix and ordinalIgnoreCase for Windows?   Feel like we're going to need that.